dolphinscheduler-dist/release-docs/licenses/LICENSE-kubernetes-client.txt

```diff
@@ -0,0 +1,201 @@
+﻿                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
+   1. Definitions.
+
+      "License" shall mean the terms and conditions for use, reproduction,
```

Comment:
For standard AL2, we don't need special declaration, so we can delete this file.